node/test/python/aws-lambda-sdk/integration.test.js

```diff
@@ -37,7 +37,8 @@ describe('Python: integration', function () {
       'success',
       {
         variants: new Map([
-          // TODO: Cover v3.6 and v3.7 once support for those is fixed
+          // TODO: Cover v3.7 once support for it is fixed
```

Comment:
Let's remove for now those TODO comments, as currently we don't see a reliable way to support Python v3.7